website/addons/base/views.py

```diff
@@ -289,6 +289,7 @@ def get_auth(auth, **kwargs):
             'callback_url': node.api_url_for(
                 ('create_waterbutler_log' if not node.is_registration else 'registration_callbacks'),
                 _absolute=True,
+                _internal=settings.DEV_MODE,
```

Comment:
Sounds like I don't have a full understanding of DEV_MODE. My assumption was that it meant local deployment, now specifically local docker deployment. It is required for local docker deployment, but I'm not sure what other scenarios we might need to cover. If production is also using the localhost aliasing idiom, then perhaps this should always be set to True. Or maybe I need to a new variable to specifically denote local docker deployment?